CRISPResso2/CRISPRessoCORE.py

```diff
@@ -2716,6 +2834,9 @@ def get_allele_row(reference_name, variant_count, aln_ref_names_str, aln_ref_sco
         ###iterate through variants
         for variant in variantCache:
             #skip variant if there were none observed
+            # check if variant is an int
+            if isinstance(variantCache[variant], int):
+                print(variant)
```

Comment:
Is this needed?

Reply:
No, that's code that I used to debug the empty string, removed